systemtest/src/main/java/io/strimzi/systemtest/TestTags.java

```diff
@@ -0,0 +1,224 @@
+     */
+    String SANITY = "sanity";
+
+    /**
+     * Tag for tests, which results are not 100% reliable on all testing environments.
+     */
+    String FLAKY = "flaky";
+
+    /**
+     * Tag for scalability tests
+     */
+    String SCALABILITY = "scalability";
+
+    /**
+     * Tag for tests containing scaling of particular component (scaling up and down)
+     */
+    String COMPONENT_SCALING = "componentscaling";
+
+    /**
+     * Tag for tests, which are working only on specific environment and we usually don't want to execute them on all environments.
+     */
+    String SPECIFIC = "specific";
+
+    /**
+     * Tag for tests, which are using NodePort.
+     */
+    String NODEPORT_SUPPORTED = "nodeport";
+
+    /**
+     * Tag for tests, which are using LoadBalancer.
+     */
+    String LOADBALANCER_SUPPORTED = "loadbalancer";
+
+    /**
+     * Tag for tests, which are using NetworkPolicies.
+     */
+    String NETWORKPOLICIES_SUPPORTED = "networkpolicies";
+
+    /**
+     * Tag for Prometheus tests
+     */
+    String PROMETHEUS = "prometheus";
+
+    /**
+     * Tag for Tracing tests
+     */
+    String TRACING = "tracing";
+
+    /**
+     * Tag for Helm tests
+     */
+    String HELM = "helm";
+
+    /**
+     * Tag for oauth tests
+     */
+    String OAUTH = "oauth";
+
+    /**
+     * Tag for recovery tests
+     */
+    String RECOVERY = "recovery";
+
+    /**
+     * Tag for tests which deploys KafkaConnector resource
+     */
+    String CONNECTOR_OPERATOR = "connectoroperator";
+
+    /**
+     * Tag for tests which deploys KafkaConnect resource
+     */
+    String CONNECT = "connect";
+
+    /**
+     * Tag for tests which deploys KafkaMirrorMaker resource
+     */
+    String MIRROR_MAKER = "mirrormaker";
+
+    /**
+     * Tag for tests which deploys KafkaMirrorMaker2 resource
+     */
+    String MIRROR_MAKER2 = "mirrormaker2";
+
+    /**
+     * Tag for tests which deploys any of KafkaConnect, KafkaConnector, KafkaMirrorMaker2
+     */
+    String CONNECT_COMPONENTS = "connectcomponents";
+
+    /**
+     * Tag for tests which deploys KafkaBridge resource
+     */
+    String BRIDGE = "bridge";
+
+    /**
+     * Tag for tests which use external Kafka clients (called from test code)
+     */
+    String EXTERNAL_CLIENTS_USED = "externalclients";
+
+    /**
+     * Tag for tests where metrics are used
+     */
+    String METRICS = "metrics";
+
+    /**
+     * Tag for tests where CruiseControl is used
+     */
+    String CRUISE_CONTROL = "cruisecontrol";
+
+    /**
+     * Tag for tests where mainly dynamic configuration is used
+     */
+    String DYNAMIC_CONFIGURATION = "dynamicconfiguration";
+
+    /**
+     * Tag for tests which contains rolling update of resource
+     */
+    String ROLLING_UPDATE = "rollingupdate";
+
+    /**
+     * Tag for tests, for Pod Security profiles set to restricted
+     */
+    String POD_SECURITY_PROFILES_RESTRICTED = "podsecurityprofiles";
```

Comment:
Same as for the `SPECIFIC` -> I never used it (and we are not using it anywhere) + it's just one class.

Reply:
Sure.